daemons/dss-notify-v2/app.py

```diff
@@ -16,7 +16,8 @@
 import dss
 from dss import Config, Replica
 from dss.logging import configure_lambda_logging
-from dss.events import get_bundle_metadata_document, get_deleted_bundle_metadata_document, record_event_for_bundle
+from dss.events import (get_bundle_metadata_document, get_deleted_bundle_metadata_document, record_event_for_bundle,
```

Comment:
is this supposed to be wrapped in parenthesis ?

Reply:
This is an alternative to using `\` to wrap lines